src/main/java/org/example/homework10/Task2/Main.java

```diff
@@ -0,0 +1,60 @@
+package org.example.homework10.Task2;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.List;
+
+public class Main {
+    public static void main(String[] args) {
+        class EmployeeProcessor {
```

Comment:
solution works correctly, but it does not reflect the idea of usage local class here. The idea was to have a class that accepts Employee in constructor and uses it as one of its fields, like
```
        class EmployeeProcessor {

            Employee containedEmployee;
            int yearsWorked;
            String fullName;

            public EmployeeProcessor(Employee containedEmployee) {
                this.containedEmployee = containedEmployee;
                // process and initialize yearsWroked
                // process and initialize fullName
            }

            @Override
            public String toString() {
           // use overriding toString to have nice input
            }
        }
```
and after it you convert list of Employee to the list of EmployeeProcessor
use comparator similar to yours but which works with EmployeeProcessor class
and print the result